### PR TITLE
Issue with k8s.io/zh/docs/tutorials/kubernetes-basics/scale/scale-intro/

### LINKED ISSUE #15147: Issue with k8s.io/zh/docs/tutorials/kubernetes-basics/scale/scale-intro/
**This is a Bug Report**

/language zh
/good-first-issue

**Problem:**

carousel not working  

**Proposed Solution:**

add carousel item class as bootstrap 4

**Page to Update:**
https://kubernetes.io/zh/docs/tutorials/kubernetes-basics/scale/scale-intro/

Related PR:
 #15129

### PR DESCRIPTION
**Pull request by mhamdisemah**
fixes #15147

### CODE REVIEW

File: content/ko/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 1]
```diff
@@ -70,7 +70,7 @@ <h2 style="color: #3771e3;">스케일링 개요</h2>
                             <img src="/docs/tutorials/kubernetes-basics/public/images/module_05_scaling1.svg">
                         </div>
 
-                        <div class="item">
+                        <div class="item ">
```
Comment (seokho-son) [inline content/ko/docs/tutorials/kubernetes-basics/scale/scale-intro.html]: @mhamdisemah , Thank you for your contribution.
However, the change for the content/**ko**/docs/tutorials/kubernetes-basics/scale/scale-intro.html seems a mistake.
In the preview (https://deploy-preview-15179--kubernetes-io-master-staging.netlify.com/ko/docs/tutorials/kubernetes-basics/scale/scale-intro/) ,
Carousel is not working for the Korean document.

Comment (mhamdisemah) [reply content/ko/docs/tutorials/kubernetes-basics/scale/scale-intro.html]: this PR is for ZH document not for KO

Comment (gochist) [reply content/ko/docs/tutorials/kubernetes-basics/scale/scale-intro.html]: @mhamdisemah :wave: This PR contains Korean content change which causes a conflict. Please resolve the conflict and remove the "language/ko" label.

**Comment by idealhack** [comment]
https://deploy-preview-15179--kubernetes-io-master-staging.netlify.com/zh/docs/tutorials/kubernetes-basics/scale/scale-intro/

/lgtm

/assign @sataqiu

**Comment by SataQiu** [comment]
Thanks @mhamdisemah 
/lgtm
/approve

**Pull request merged**